src/peer-routing.js

```diff
@@ -71,7 +78,7 @@ class PeerRouting {
   }
 
   /**
-   * Iterates over all peer routers in series to find the given peer.
+   * Iterates over all peer routers in parallel to find the given peer.
```

Comment:
Same as content-routing with the constructor comment

Reply:
Done!